src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj

```diff
@@ -3,7 +3,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);CA2249</NoWarn>
     <DefineConstants>$(DefineConstants);REGISTRY_ASSEMBLY</DefineConstants>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;$(NetFrameworkCurrent)-Windows_NT;netstandard2.0-Windows_NT;netstandard2.0-Unix;netstandard2.0;net461-Windows_NT</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;$(NetFrameworkCurrent)-Windows_NT;netstandard2.0-Windows_NT;netstandard2.0-Browser;netstandard2.0-Unix;netstandard2.0;net461-Windows_NT</TargetFrameworks>
```

Comment:
I don't expect registry to be supported in browser, should we throw PNSE for this one? cc: @marek-safar ?

Reply:
The unix implemention is based on filesystem, so it could work.  Given this is just a package I'm not sure if we should go out of our way to do that by adding a browser config.  I wonder if we refactored the current implementation to be RID-less rather than unix?  That way the package doesn't grow but we still support both runtimes.

Reply:
Appears that registry made its way into 3.2.  Not sure how well it works.

Reply:
It'd be potential breaking change but we need to test what from Unix version works in Browser as the sandbox filesystem is a bit more restrictive than Unix.